integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/IntegrationProvisioningService.java

```diff
@@ -179,9 +179,14 @@ private void disableApplication(Application application) {
     application.setEnabled(Boolean.FALSE);
     application.setVisible(Boolean.FALSE);
 
-    applicationService.updateAppSettings(application);
+    boolean updated = applicationService.updateAppSettings(application);
 
-    LOGGER.info("Application {} disabled\n", application.getId());
+    if (updated) {
+      application.setState(ApplicationState.REMOVED);
+      LOGGER.info("Application {} disabled\n", application.getId());
+    } else {
+      application.setState(ApplicationState.SKIPPED);
```

Comment:
Just add a log info here "Application {} skipped, as it is already not provisioned on the backend."